### PR TITLE
Update middleware to match helmet.js default values

### LINKED ISSUE #42: Stricter default value
Should we enforce stricter default value like [helmetjs](https://github.com/helmetjs/helmet#how-it-works) ?
```
Content-Security-Policy: default-src 'self';base-uri 'self';block-all-mixed-content;font-src 'self' https: data:;frame-ancestors 'self';img-src 'self' data:;object-src 'none';script-src 'self';script-src-attr 'none';style-src 'self' https: 'unsafe-inline';upgrade-insecure-requests
X-DNS-Prefetch-Control: off
Expect-CT: max-age=0
X-Frame-Options: SAMEORIGIN
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Download-Options: noopen
X-Content-Type-Options: nosniff
X-Permitted-Cross-Domain-Policies: none
Referrer-Policy: no-referrer
X-XSS-Protection: 0
```

### PR DESCRIPTION
**Pull request by gaby**
- Updated the default values of the middleware to match the ones from `Helmet.js`.
- Given that these may be considered "breaking changes", it may require bumping the minor version of the middleware.
- Note, the tests are not testing the HSTS age headers, since that requires running a HTTPS server.

Source for default values: https://github.com/helmetjs/helmet#quick-start

Fixes #42

### CODE REVIEW

**Comment by gaby** [comment]
I plan to add to add 2-3 more tests tomorrow. For example we are not testing setting custom values, setting real-values other than the default ones.

**Comment by gaby** [comment]
This one should be ready for review @ReneWerner87

**Comment by ReneWerner87** [comment]
Ok, tomorrow

File: main.go [hunk 1]
```diff
@@ -80,6 +128,32 @@ func New(config ...Config) fiber.Handler {
 		if cfg.XFrameOptions != "" {
 			c.Set(fiber.HeaderXFrameOptions, cfg.XFrameOptions)
 		}
+		if cfg.CrossOriginEmbedderPolicy != "" {
+			c.Set("Cross-Origin-Embedder-Policy", cfg.CrossOriginEmbedderPolicy)
```
Comment (efectn) [inline main.go:132]: Should we add new consts for these headers if they are not exists?

Comment (gaby) [reply main.go:132]: I'm trying to make it match what `helmet.js` does. #42 suggested stricter values, but while doing so I noticed we are missing some headers that `helmet.js` supports.

Comment (efectn) [reply main.go:132]: I think we can add them to fiber core 🤔

Comment (gaby) [reply main.go:132]: @efectn Yeah, I feel like this middleware should be part of the core. It's quite simple implementation wise.

**Review by efectn — APPROVED** [review]

**Pull request merged**